pages/spicedb/concepts/datastores.mdx

```diff
@@ -30,6 +30,44 @@ spicedb migrate head \
 
 For more information, refer to the [Datastore Migration documentation](/spicedb/concepts/datastore-migrations).
 
+## Read replicas for Postgres and MySQL
+
+SpiceDB can reference read-only replicas of permissions data stored in Postgres and MySQL that respect consistency requirements by detecting if a given revision has been replicated from the
+primary datastore before reading from the replica.
+If the requested revision is not found in the replica, the request will be redirected to the primary datastore.
+
+Strict read mode (available for Postgres only) does not check replicas for requested revisions before reading from them; instead, a revision check is inserted into the SQL for each read.
+This mode is useful for deploying replicas behind a load balancer than can transparently handle these requests.
+To reduce overhead of using read replicas, enable caching on CheckRevision calls to avoid checking the requested revision if the last checked revision is at least as fresh as the one
+requested.
```

Comment:
I don't think we need to describe these implementation details here. What we do need to describe is the following:
- each one of the replica host DNS can be listed one by one
- Additionally, SpiceDB supports Postgres replicas behind a load-balancer. MySQL replicas cannot be exposed to SpiceDB via a load balancer, and the individual host DNS must be listed. ⚠️ Having SpiceDB connect to MySQL replicas behind a load-balancer can lead to consistency issues.

```suggestion

```

Reply:
(see comment below- incorporated feedback by deleting and reorganizing)